src/DtoGenerator/DtoGenerator.Logic/Model/PropertyMetadata.cs

```diff
@@ -21,6 +22,7 @@ public class PropertyMetadata
         public bool IsInherited { get; set; }
 
         public EntityMetadata RelationMetadata { get; set; }
+        public SyntaxList<AttributeListSyntax> AttributesList { get; set; }
```

Comment:
I would prefer not using SyntaxList here, but rather List<string> AttributesList. I'll be throwing out "SyntaxNode" property as well since it is not used anywhere (probably some leftover of previous versions).